client/client.ts

```diff
@@ -245,10 +253,18 @@ async function gameLoop() {
  */
 socket.on("connect", async () => {
     console.log("Server connection established");
+    PLAYER.socketId = socket.id;
```

Comment:
hmm i don't track- why do we need to save socketId client side?

shouldn't enclave be the one to track this? otherwise players can lie about which socket they're plugging into

Reply:
you right, not needed and prone to attack